core/templates/pages/learner-group-pages/edit-group/learner-group-learners-progress.component.html

```diff
@@ -0,0 +1,95 @@
+          {{'I18N_LEARNER_GROUP_VIEW_DETAILS' | translate}}
+        </div>
+      </div>
+      <div class="learner-progress-summary-info">
+        <span class="warning">Struggling with {{getStrugglingSubtopicsCountOfLearner(idx)}} skills
+          <span class="divider"> | </span>
+        </span>
+        <span>Completed {{getCompletedStoriesCountByLearner(idx)}} stories</span>
+      </div>
+    </div>
+    <div *ngIf="!learnerProgress.isProgressSharingTurnedOn" class="learner-progress-summary">
+      <div class="learner-progress-summary-header">
+        <div class="learner-info">
+          <img [src]="getProfileImageDataUrl(learnerProgress.profilePictureDataUrl)">
+          <div>{{learnerProgress.username}}</div>
+        </div>
+        <div class="permissions-message gray-message">
+          {{'I18N_LEARNER_GROUP_PERMISSION_NOT_GIVEN' | translate}} &nbsp;
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
+            <path d="M10 2.00391C5.58879 2.00391 2 5.38879 2 9.54937C2 13.7099 5.58879 17.0948 10 17.0948C14.4112 17.0948 18 13.7099 18 9.54937C18 5.38879 14.4112 2.00391 10 2.00391ZM10 0.00390625C15.5229 0.00390625 20 4.27756 20 9.54937C20 14.8212 15.5229 19.0948 10 19.0948C4.47715 19.0948 0 14.8212 0 9.54937C0 4.27756 4.47715 0.00390625 10 0.00390625Z" fill="#757575"/>
+            <path d="M10.7119 12.0879H9.44043C9.44499 11.6504 9.48372 11.2926 9.55664 11.0146C9.63411 10.7321 9.75944 10.4746 9.93262 10.2422C10.1058 10.0098 10.3359 9.74544 10.623 9.44922C10.8327 9.23503 11.0241 9.03451 11.1973 8.84766C11.375 8.65625 11.5186 8.45117 11.6279 8.23242C11.7373 8.00911 11.792 7.74251 11.792 7.43262C11.792 7.11816 11.735 6.84701 11.6211 6.61914C11.5117 6.39128 11.3477 6.21582 11.1289 6.09277C10.9147 5.96973 10.6481 5.9082 10.3291 5.9082C10.0648 5.9082 9.81413 5.95605 9.57715 6.05176C9.34017 6.14746 9.14876 6.29557 9.00293 6.49609C8.8571 6.69206 8.7819 6.94954 8.77734 7.26855H7.5127C7.52181 6.75358 7.64941 6.31152 7.89551 5.94238C8.14616 5.57324 8.4834 5.29069 8.90723 5.09473C9.33105 4.89876 9.80501 4.80078 10.3291 4.80078C10.9079 4.80078 11.4001 4.9056 11.8057 5.11523C12.2158 5.32487 12.528 5.62565 12.7422 6.01758C12.9564 6.40495 13.0635 6.86523 13.0635 7.39844C13.0635 7.80859 12.9792 8.18685 12.8105 8.5332C12.6465 8.875 12.4346 9.19629 12.1748 9.49707C11.915 9.79785 11.6393 10.085 11.3477 10.3584C11.097 10.5908 10.9284 10.8529 10.8418 11.1445C10.7552 11.4362 10.7119 11.7507 10.7119 12.0879ZM9.38574 14.2549C9.38574 14.0498 9.44954 13.8766 9.57715 13.7354C9.70475 13.5941 9.88932 13.5234 10.1309 13.5234C10.377 13.5234 10.5638 13.5941 10.6914 13.7354C10.819 13.8766 10.8828 14.0498 10.8828 14.2549C10.8828 14.4508 10.819 14.6195 10.6914 14.7607C10.5638 14.902 10.377 14.9727 10.1309 14.9727C9.88932 14.9727 9.70475 14.902 9.57715 14.7607C9.44954 14.6195 9.38574 14.4508 9.38574 14.2549Z" fill="#757575"/>
```

Comment:
Ditto my earlier comment, this line is too long.

Reply:
Fixed!